CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
-# 2.2.0
+# 2.3.0
+
+### :hammer_and_wrench: Improvements
+* Add `httpClientBuilder` attribute to `QldbDriverBuilder` to allow customized HTTP client.
+    * The following code demonstrates how to use it:
+        ```
+        QldbSessionClientBuilder sessionClientBuilder = QldbSessionClient.builder();
+        
+        QldbDriverBuilder builder = QldbDriver
+            .builder()
+            .ledger(ledger);
+            .maxConcurrentTransactions(poolLimit)
+            .transactionRetryPolicy(TransactionRetryPolicy.builder().maxNumberOfRetries(retryLimit).build());
+            .sessionClientBuilder(sessionClientBuilder)
+            .httpClientBuilder(ApacheHttpClient.builder().maxConnections(maxConnections))
+            .build();
+        ```
+  * Note: The maximum connections set in the HTTP client should be equal or greater than the `maxConcurrentTransactions`
+    to avoid connection contentions and poor performance.
```

Comment:
nit: `leading to poor performance` sounds better

Reply:
Updated.